code/src/Components/TopRatedList.js

```diff
@@ -0,0 +1,202 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+
+
+export const TopRatedList = ({ movieList }) => {
+	const [filteredMovieList, setfilteredMovieList] = useState(movieList)
+	const [filter, setFilter] = useState()
+	const [sorting, setSorting] = useState()
+
+	// ADDED second useEffect to filter movies based on genre
```

Comment:
Maybe this filter-function could be imported to the MovieDetails as well? They look to be about the same.